### DISCUSSION TITLE
Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by msonowal** [Schweinepriester/github-profile-achievements#235] [General]
someone create a discuss  Galaxy Brain Achievement I will accept this answer to help get the badge

**Comment by dookki** [comment]
oke

**Comment by mezotv** [comment]
Amazing!

**Comment by sinalalebakhsh** [comment]
thank you https://github.com/msonowal

**Reply by msonowal** [reply]
can you create a question in the discussion so that I can answer and you can accept

**Comment by theyasub** [comment]
Hello everyone I have one question 
how I get Galaxy brain achievement

**Reply by mezotv** [reply]
You need to answer 2 discussions

**Reply by sinalalebakhsh** [reply]
How build 2 ?

**Reply by Sara-Mediouni** [reply]
and the pull shark one?

**Reply by mezotv** [reply]
> and the pull shark one?

You can get that by opening 2 pull requests that got merged

**Comment by EeeasyCode** [comment]
hello everyone!
Im just study to programing
give me a any advice

**Comment by Sayrix** [comment]
Answering!
